src/vm/amd64/cgencpu.h

```diff
@@ -268,6 +268,14 @@ struct CalleeSavedRegistersPointers {
 #define THIS_kREG kRCX
 #endif
 
+#ifdef PLATFORM_UNIX
```

Comment:
Can you please put it into the previous ifdef? Also, the UNIX_AMD64_ABI is a better ifdef for this. And last, x86 uses ARGUMENT_REG1 notation, so it would be nice to use the ARGUMENT_kREG1 as I've suggested before.